src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.domain;
+
+import racingcar.exception.RacingCarErrorMessage;
+
+public class Car {
+
+  public static final int MIN_RUNNABLE_FUEL = 4;
+  private final String name;
+  private int position = 0;
+  private int fuel = 0;
+
+  public Car(final String name) {
+    validateName(name);
+    this.name = name;
+  }
+
+  private void validateName(String name) {
+    if (name.length() < 1 || name.length() > 5) {
+      throw new IllegalStateException(RacingCarErrorMessage.CAR_NAME.getMessage());
+    }
+  }
+
+  public void run() {
+    if (isRunnable(fuel)) {
+      this.position++;
+    }
+    fuel = 0;
+  }
+
+  public void fillInFuel(final int fuel) {
+    this.fuel = fuel;
+  }
+
+  private boolean isRunnable(final int fuel) {
+    return fuel >= MIN_RUNNABLE_FUEL;
+  }
```

Comment:
fuel 을 인스턴스 필드로 가지고 있다면 파라미터를 통해 주입 받을 필요 없이 아래와 같이 표현 할 수 있을 것 같아요.
```suggestion
  private boolean isRunnable() {
    return this.fuel >= MIN_RUNNABLE_FUEL;
  }
```